.gitignore

```diff
@@ -63,3 +63,6 @@ Icon
 Network Trash Folder
 Temporary Items
 .apdisk
+
+# GraphQL Schema
+dashx/main/graphql/com/dashx/schema.json
```

Comment:
You added to gitignore, but the file didn't get removed. You should delete the file -> commit -> then add to gitignore. (OR) you should use `git rm ...` to remove it from the index.